pkg/rbd/controllerserver.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/pborman/uuid"
```

Comment:
should use `k8s.io/apimachinery/pkg/util/uuid` instead of `github.com/pborman/uuid`, the former tries to avoid generate identical UUIDs if the time interval is quick enough. (See https://github.com/kubernetes/apimachinery/blob/master/pkg/util/uuid/uuid.go#L34)

Reply:
thanks, will change..